Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -15,10 +16,32 @@ class DocumentViewModel: ObservableObject {
         guard let baseFileUrl = URL(string: model.baseFileUrl) else {
             return nil
         }
-
         self.model = model
         self.pdfDocument = PdfViewModel(baseFileUrl: baseFileUrl)
         self.annotations = model.annotations.map { AnnotationViewModel(model: $0, document: self) }
+        setUpSubscribersForAllAnnotations()
+    }
+
+    private func setUpSubscribersForAllAnnotations() {
+        for annotation in annotations {
+            annotation.$isInFocus.sink(receiveValue: { [weak self] isInFocus in
+                if isInFocus {
+                    self?.setAllOtherAnnotationsOutOfFocus(except: annotation)
+                }
+            }).store(in: &cancellables)
+        }
+    }
+
+    func setAllAnnotationsOutOfFocus() {
+        for annotation in annotations {
+            annotation.outOfFocus()
+        }
+    }
+
+    private func setAllOtherAnnotationsOutOfFocus(except annotationInFocus: AnnotationViewModel) {
+        for annotation in annotations where annotation.model.id != annotationInFocus.model.id {
```

Comment:
```suggestion
        for annotation in annotations where annotation.id != annotationInFocus.id {
```

Could we have a computed property in `AnnotationViewModel`?
```
var id: UUID {
    model.id
}
```

Reply:
Ok done